src/Raven.Server/Smuggler/Documents/SmugglerBase.cs

```diff
@@ -713,6 +714,13 @@ await foreach (var index in _source.GetIndexesAsync())
                         continue;
                     }
 
+                    if (_options.AuthorizationStatus == AuthorizationStatus.ValidUser &&
```

Comment:
This will filter things out silently, which can be _super_ confusing to the user.

Reply:
We're adding an error to the import log.

Reply:
In the UI, is that clearly visible?

Reply:
![image](https://github.com/ravendb/ravendb/assets/18046191/87db4396-6174-47a6-b26c-2a6774c2e4a8)